src/simplewallet/simplewallet.cpp

```diff
@@ -873,6 +1482,14 @@ simple_wallet::simple_wallet()
   m_cmd_binder.set_handler("password", boost::bind(&simple_wallet::change_password, this, _1), tr("Change wallet password"));
   m_cmd_binder.set_handler("payment_id", boost::bind(&simple_wallet::payment_id, this, _1), tr("Generate a new random full size payment id - these will be unencrypted on the blockchain, see integrated_address for encrypted short payment ids"));
   m_cmd_binder.set_handler("fee", boost::bind(&simple_wallet::print_fee_info, this, _1), tr("Print information about fee and current transaction backlog"));
+  m_cmd_binder.set_handler("prepare_multisig", boost::bind(&simple_wallet::prepare_multisig, this, _1), tr("Export data needed to create a multisig wallet"));
+  m_cmd_binder.set_handler("make_multisig", boost::bind(&simple_wallet::make_multisig, this, _1), tr("Turn this wallet into a multisig wallet"));
+  m_cmd_binder.set_handler("finalize_multisig", boost::bind(&simple_wallet::finalize_multisig, this, _1), tr("Turn this wallet into a multisig wallet, extra step for N-1/N wallets"));
+  m_cmd_binder.set_handler("export_multisig_info", boost::bind(&simple_wallet::export_multisig, this, _1), tr("Export multisig info for other participants"));
+  m_cmd_binder.set_handler("import_multisig_info", boost::bind(&simple_wallet::import_multisig, this, _1), tr("Import multisig info from other participants"));
```

Comment:
These command names are inconsistent with usage message: `usage: export_multisig <filename>`. IMO those without `_info` seem just fine as they're shorter and without any ambiguity.

Reply:
The reason for the _info is that when adding the raw export function (export_raw_multisig_tx), I found myself always doing export_mTAB, ending up with export_multisig, then adding filename and enter, and ending up using the wrong command. So I thought adding _info would make it clear it was the wrong command. Maybe it's just because I'd become used to that command though, and a new user would not...

Reply:
I'm fine with the current command name when there's a reason. So it's the error message that needs to be corrected.

Reply:
Might be better to have export_multisig_info and export_multisig_raw_tx. That way, when you tab, you get both and are forced to choose one explicitely.